### ISSUE TITLE
Duplicate and confusing search results

### ISSUE THREAD

**Issue by Krinkle** [eslint/eslint.github.io#459]
Searching for the word "disable":

<img width="595" alt="screen shot 2018-03-16 at 16 25 01" src="https://user-images.githubusercontent.com/156867/37548667-b326dda4-2936-11e8-8008-45f131905c1d.png">

These results have two issues:

1. I don't know what they represent. (It doesn't say "rule" or "no-lonely-if", which is what the doc snippets are from.)
2. They all look the same. (It doesn't say which release the page belongs to.)

The results should instead contain something like:

1. ( "rule", "no-lonely-if", "**v0.24.1**" ) [..]
2. ( "rule", "no-lonely-if", "**v1.10.3**" ) [..]
3. [..]

etc.

**Comment by ilyavolodin** [comment]
Thanks for reporting this. We've head a bit of trouble with today's release (looks like Github updated some of the dependencies) so it's possible that this issue is related to that. I'll try to figure out a way to clean this up.

**Comment by ilyavolodin** [comment]
@pixelastic Sorry to bother you, but as I mentioned we ran into some problems earlier today with Github pages, and had to update some dependencies. Since then, it looks like Algolia is not able to build a new index, because it's claiming that it can't push changes to {index_name}_tmp index. Algolia API key hasn't been touched, so I assume the error is due to something else. Would you be able to help us out with this problem?

**Comment by ilyavolodin** [comment]
Ah, I see what the issue is: Record quota exceeded, change plan or delete records. @pixelastic Any chance for slightly bumping our quota?

**Comment by pixelastic** [comment]
Fixed!

**tl;dr:** I upgraded your maximum quota from 100k records to 500k records. It should give you more breathing room.

**Longer version:**
If you get the chance, I would also suggest either:

- Upgrading to the new version of the plugin (now named [jekyll-algolia](https://community.algolia.com/jekyll-algolia/)). It has a much smarter update mechanism that consumes less operations, is much faster to run and overall provides a better DX.

- Replace the Jekyll plugin with our [DocSearch](https://community.algolia.com/docsearch/) crawler. This will run every 24h on our infra, scraping webpages from your website. You'll never have to worry about overquota issues as it's free forever.

Even if I'm the author of the jekyll-algolia plugin, I have to say that DocSearch is a better choice for documentation websites today as it "just works" out of the box without you having to deal with custom configs and external Travis pipeline.

Cheers,

**Comment by Krinkle** [comment]
Not sure what it looked like before, but to me it still looks the same:

<img width="1091" alt="screen shot 2018-03-20 at 11 15 10" src="https://user-images.githubusercontent.com/156867/37674377-00aecb58-2c30-11e8-84fe-917f6a73e198.png">

**Comment by pixelastic** [comment]
@ilyavolodin You might have to trigger a rebuild (either `jekyll algolia push` locally, or pushing to GitHub pages and letting Travis handle it from there)

**Comment by ilyavolodin** [comment]
@pixelastic Awesome! Thank you very much for doing this. We will be doing a patch release later tonight, so that should re-run the job that would index our site. Again, appreciate your help! I will take a look at DocSearch and try to set it up, sounds like it should make our operations a bit smoother.
